src/test/java/org/jabref/logic/integrity/ValidCitationKeyCheckerTest.java

```diff
@@ -0,0 +1,30 @@
+package org.jabref.logic.integrity;
+
+import java.util.Optional;
+import java.util.stream.Stream;
+
+import org.jabref.logic.l10n.Localization;
+
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class ValidCitationKeyCheckerTest {
+
+    private final ValidCitationKeyChecker checker = new ValidCitationKeyChecker();
+
+    @ParameterizedTest
+    @MethodSource("provideCitationKeys")
+    void citationKeyValidity(Optional optionalArgument, String citationKey, String errorMessage) {
```

Comment:
`errorMessage` is unsued now - remove it, please

```suggestion
    void citationKeyValidity(Optional optionalArgument, String citationKey) {
```

Reply:
Please also remove the variables from the call